node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerNetworking.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.node.admin.docker;// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+import com.yahoo.config.provision.NodeType;
+
+/**
+ * The types of network setup for the Docker containers.
+ *
+ * @author hakon
+ */
+public enum DockerNetworking {
+    /** Each container has an associated macvlan bridge. */
+    MACVLAN,
+
+    /** Network Prefix-Translated networking. */
+    NPT,
+
+    /** A host running a single container in the host network namespace. */
+    HOST_NETWORK;
+
+    public static DockerNetworking from(String cloud, NodeType nodeType, boolean hostAdmin) {
```

Comment:
This should just be set in hosted code when creating the `Environment`?

Reply:
Possibly